packages/components/ng-ovh-payment-method/src/legacy/payment-method-legacy.js

```diff
@@ -1,9 +1,9 @@
-import filter from 'lodash/filter';
-import flatten from 'lodash/flatten';
-import get from 'lodash/get';
-import has from 'lodash/has';
-import map from 'lodash/map';
-import startCase from 'lodash/startCase';
+import { filter } from 'lodash-es';
```

Comment:
yes it could be merged in a single import but i used a script to batch rename imports and it's not handled for now :)